### DISCUSSION TITLE
S-HOOK 3차 데모데이 회고

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#73] [General]
# S-HOOK 3차 데모데이 회고

어느덧 2주 후면 4차 데모데이고, 레벨3가 끝난다. S-HOOK 프로젝트를 진행한 후로 시간이 많이 흐른 지금 프로젝트 회고를 작성해보려고 한다.     S-HOOK 이란 스플릿의 아이디어로 시작된 S-HOOK은 주제를 보자마자 하고 싶었던 프로젝…

https://vero.wiki/shook-third-demoday-retrospect/

**Comment by splitCoding** [comment]
베로 글을 왜캐 잘씀!!👍 
우리 팀원들과 같이 해서 행복하다아~ 😁 ( 글이 뭔가 뭉클함 늙어서 그런 듯 )

**Reply by Cyma-s** [reply]
팀장님 충성충성 ^^7
